weed/shell/command_volume_tier_move.go

```diff
@@ -64,6 +64,7 @@ func (c *commandVolumeTierMove) Do(args []string, commandEnv *CommandEnv, writer
 	applyChange := tierCommand.Bool("force", false, "actually apply the changes")
 	ioBytePerSecond := tierCommand.Int64("ioBytePerSecond", 0, "limit the speed of move")
 	replicationString := tierCommand.String("toReplication", "", "the new target replication setting")
+	disableSrcParallelLimit := tierCommand.Bool("disableSrcParallelLimit", false, "limit each src server to only one job")
```

Comment:
how about change this option to the inflight job count limit?

Reply:
Sorry for the late reply. The problem is there's actually two different bottleneck during volume.tier.move, so we actually need both read concurrency limit and write concurrency limit.
For example moving from ssd to hdd, we would like limiting the write concurrency to hdd, but not limiting read concurrency from ssd.

Probably we can change -disableSrcParallelLimit to -concurrency & -srcConcurrency, the -concurrency limits how many copy jobs can execute in parallel, and -srcConcurrency limits how many volumes can be read concurrently from each source volume server.

If you feel good for above change, I'll start to implement it :)

Reply:
Sounds good!